automated_analysis.py

```diff
@@ -465,4 +460,47 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         fig.write_image(f"{automated_analysis_output_dir}/graphs/{plan.raw_field}_by_gender_normalised.png",
                         scale=IMG_SCALE_FACTOR)
 
+    # Export safe to share raw messages for each episode to share with WUSC
```

Comment:
Maybe add a brief description of how e.g. "Messages are safe to share if they have been reviewed and do not contain a 'DNS' (do not share) label"

Reply:
updated thanks